gtk/vendor.md

```diff
@@ -2539,7 +2490,7 @@ react to the text direction.
 
 [`ArrowExt`](trait.ArrowExt.html), [`MiscExt`](trait.MiscExt.html), [`WidgetExt`](trait.WidgetExt.html), [`glib::object::ObjectExt`](../glib/object/trait.ObjectExt.html), [`BuildableExt`](trait.BuildableExt.html), [`WidgetExtManual`](prelude/trait.WidgetExtManual.html), [`BuildableExtManual`](prelude/trait.BuildableExtManual.html)
 <!-- trait ArrowExt -->
-`[Deprecated since 3.14]` Trait containing all `Arrow` methods.
+Trait containing all `Arrow` methods.
```

Comment:
Seems cargo doc still add totally deprecated docs, this is wrong.

Reply:
* `gir doc`

Reply:
Indeed, gir PR incoming I assume!